include/global.h

```diff
@@ -391,11 +391,11 @@ struct BattleFrontier
     /*0xEBA*/ u16 field_EBA;
     /*0xEBC*/ u32 battlesCount;
     /*0xEC0*/ u16 field_EC0[16];
-    /*0xEE0*/ u8 field_EE0;
+    /*0xEE0*/ u8 trainerFlags;
     /*0xEE1*/ u8 opponentName[2][PLAYER_NAME_LENGTH + 1];
     /*0xEF1*/ u8 field_EF1[2][4];
-    /*0xEF9*/ u8 field_EF9_0:7;
-    /*0xEF9*/ u8 field_EF9_1:1;
+    /*0xEF9*/ u8 neverRead:7;
```

Comment:
I'd prefer `unk*` rather than `neverRead`, in general.

Reply:
Alright, I'll update that. I don't have a strong preference one way or the other, though I prefer `neverRead` slightly because it contains at least a little information.